integration/test_hnsw_allow_replace_deleted.py

```
@@ -184,3 +185,111 @@ def test_index_add_after_rdb_load_with_deleted_elements(self):
+            lambda: hnsw_index.info(client).num_docs, 2
+        )
+
+        # Pause the next modify between its markDelete and addPoint.
+        assert client.execute_command(
+            "FT._DEBUG", "PAUSEPOINT", "SET",
+            "hnsw_modify_between_delete_and_add") == b"OK"
+
+        # Modify doc:0. markDelete runs on its slot, and then the thread parks
+        # at the pausepoint before addPoint.
+        vec0_new = struct.pack('<4f', 10.0, 20.0, 30.0, 40.0)
+        update_thread, _, update_err = run_in_thread(
```

Ruff (0.16.1)

[warning] Unpacked variable `update_err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)